### PR TITLE
Desert Dino Lasers and Scroll Behind Bar

### PR DESCRIPTION
**Pull request by Abnormal202**
## Description
In the latest groundbreaking update, a remarkable enhancement has been introduced to the prehistoric experience within our virtual realm—an innovative laser ability has been seamlessly integrated into the skeletal structure of dinosaurs. This awe-inspiring addition not only elevates the overall gameplay dynamics but also provides an immersive and futuristic touch to the ancient world, allowing players to unleash the power of cutting-edge technology upon the timeless bones of these majestic creatures.

Furthermore, in the realm of social interaction and libation indulgence, a subtle yet significant transformation has taken place at the heart of the virtual tavern. A meticulously crafted scroll has found its new home discreetly behind the counter, adding an air of mystery and intrigue to the already vibrant atmosphere. This subtle addition invites players to embark on a quest for knowledge or perhaps unveil hidden secrets, as the scroll promises to be a repository of untold stories and undiscovered lore.

However, it is essential to note that the full potential of this virtual tapestry remains tantalizingly out of reach for the time being. As of the current state of affairs, accessing the scroll necessitates the utilization of the noclip feature, a temporary workaround due to the absence of mirage functionality. The intricate dance between mirage tiles and the scroll effect has encountered a minor hiccup, resulting in a temporary inability to seamlessly traverse the mystical boundaries that separate our players from the enigmatic contents concealed within the scroll.

Theoretically, the introduction of this sophisticated scroll should initiate a cascade of effects, transforming the very fabric of the virtual environment. For instance, the hitherto impervious safe, presumed to be susceptible to the laser-induced metamorphosis, stands as a testament to the potential consequences of this digital alchemy. Alas, the intricacies of this transformative process remain veiled, as the current impediment in the synchronization between the scroll effect and mirage tiles has precluded a thorough examination of the safe's response to the impending scroll-induced metamorphosis.

In conclusion, while the tantalizing prospect of a melting safe teases our collective imagination, the current technical constraints have relegated this potential phenomenon to the realm of speculation. As the virtual landscape continues to evolve and the development team diligently works to unravel the mysteries entwined within the scroll and mirage intricacies, we can only anticipate the grand unveiling of a seamless and awe-inspiring fusion between the ancient and the futuristic in our cherished virtual realm.

## Related Task
task?

## How Has This Been Tested?
lots of ES 7, followed by moving tiles every which way to see if lasers are working correct.

## Screenshots (if appropriate):
image

### CODE REVIEW

File: Slider/Assets/Scripts/Sliders/Grid/AreaSTiles/MirageSTileManager.cs [hunk 1]
```diff
@@ -60,25 +60,28 @@ public void DisableMirage(int islandId = -1)
...
-                tileIdToPosition[mirageTileGridPosition] = tileId + 1;
+                //tileIdToPosition[mirageTileGridPosition] = tileId + 1;
+                tileIdToPosition[mirageTileGridPosition] = alphabet[tileId];
             }
         }
 
         return tileIdToPosition;
     }
 
+    private static string alphabet = "abcdefghijklmnopqrstuvwxyz";
```
Comment (ChaseOB) [inline Slider/Assets/Scripts/Sliders/Grid/AreaSTiles/MirageSTileManager.cs:83]: nits: 
should we use capital letters like we do for magitech past tiles?
you don't need the full alphabet
you can do int to char conversion w/o a map

**Review by randomerz — COMMENTED** [review]
theres a lot of bugs but just fix the comments and ill merge /o\
